landowner.cc

```diff
@@ -0,0 +1,133 @@
+#include <iostream>
+#include "landowner.h"
+#include <vector>
+#include <iterator>
+#include <algorithm>
+#include <cstdlib>
+#include <ctime>
+
+Landowner::Landowner(){
+  SetName("Default landowner");
+  SetSex("Secret");
+  SetExp(0);
+  SetGold(1000);
+  std::cout<<"Landowner()"<<std::endl;
+}
+Landowner::Landowner(std::string name):m_Name(name)
+{
+  SetSex("Secret");
```

Comment:
No need to call member functions these would be faster to assign using : gold_{1000}, ...